specification/domainservices/resource-manager/Microsoft.AAD/stable/2022-09-01/domainservices.json

```diff
@@ -0,0 +1,1151 @@
+          "x-ms-enum": {
+            "name": "NtlmV1",
+            "modelAsString": true
+          }
+        },
+        "tlsV1": {
+          "type": "string",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "default": "Enabled",
+          "description": "A flag to determine whether or not TlsV1 is enabled or disabled.",
+          "x-ms-enum": {
+            "name": "TlsV1",
+            "modelAsString": true
+          }
+        },
+        "syncNtlmPasswords": {
+          "type": "string",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "default": "Enabled",
+          "description": "A flag to determine whether or not SyncNtlmPasswords is enabled or disabled.",
+          "x-ms-enum": {
+            "name": "SyncNtlmPasswords",
+            "modelAsString": true
+          }
+        },
+        "syncKerberosPasswords": {
+          "type": "string",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "default": "Enabled",
+          "description": "A flag to determine whether or not SyncKerberosPasswords is enabled or disabled.",
+          "x-ms-enum": {
+            "name": "SyncKerberosPasswords",
+            "modelAsString": true
+          }
+        },
+        "syncOnPremPasswords": {
+          "type": "string",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "default": "Enabled",
+          "description": "A flag to determine whether or not SyncOnPremPasswords is enabled or disabled.",
+          "x-ms-enum": {
+            "name": "SyncOnPremPasswords",
+            "modelAsString": true
+          }
+        },
+        "kerberosRc4Encryption": {
+          "type": "string",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "default": "Enabled",
+          "description": "A flag to determine whether or not KerberosRc4Encryption is enabled or disabled.",
+          "x-ms-enum": {
+            "name": "KerberosRc4Encryption",
+            "modelAsString": true
+          }
+        },
+        "kerberosArmoring": {
+          "type": "string",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "default": "Disabled",
+          "description": "A flag to determine whether or not KerberosArmoring is enabled or disabled.",
+          "x-ms-enum": {
+            "name": "KerberosArmoring",
+            "modelAsString": true
+          }
+        },
+        "ldapSigning": {
+          "type": "string",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "default": "Disabled",
+          "description": "A flag to determine whether or not LdapSigning is enabled or disabled.",
+          "x-ms-enum": {
+            "name": "LdapSigning",
+            "modelAsString": true
+          }
+        },
+        "channelBinding": {
```

Comment:
>channelBinding [](http://example.com/codeflow?start=9&length=14)

Potentially something similar here if it makes sense, e.g. bindingType: { None, Channel } (or a better version you might come up with).

Reply:
There aren't any different binding options.  Channel Binding is either on or off.

Reply:
@mentat9 same as above, let me know if you are ok with these names.  We have discussed them internally and have aproved them.